examples/how_to/custom/js/child.py

```diff
@@ -0,0 +1,17 @@
+import param
+
+import panel as pn
+
+from panel.custom import JSComponent, View
+
+
+class Example(JSComponent):
+
+    child = View()
```

Comment:
I think the name View can be misleading. How will you explain that there is also a view on the JavaScript side, But they are not related?

I think Child/ Children is a better name and related to the children name on the JavaScript side.

Reply:
Indeed, I'm okay with Child/Children but don't love it.

Reply:
Renamed to `Child`/`Children`.